registry.yaml

```
@@ -280,7 +280,7 @@
     - k6/x/sql/driver/azuresql
   categories:
     - data
-  constraints: ">=0.1.0"
+  constraints: "=0.1.0, =0.1.1"
```

This won't work because no version can satisfy these two constraints simultaneously.  Use `versions` instead:
```suggestion
  versions: 
   - v0.1.0
   - v0.1.1
```